base_bank_from_iban/models/res_partner_bank.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2017 Tecnativa - Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3).
+
+from odoo import api, models
+from odoo.addons.base_iban.models.res_partner_bank import \
+    normalize_iban, pretty_iban, _map_iban_template
+
+
+class ResPartnerBank(models.Model):
+    _inherit = 'res.partner.bank'
+
+    @api.multi
+    @api.onchange('acc_number')
```

Comment:
According to the code, this should depend on `acc_type` too

Reply:
It shouldn't be possible to change acc_type without changing acc_number

Reply:
The problem is that we don't know if `acc_type` is computed before or after calling this onchange, so it's better to include that field as dependency here for not depending on the ORM implementation.